### ISSUE TITLE
[CoE Starter Kit - QUESTION] New applications and creators are missing in COE App - Admin - App Permission Centre - Admin - Sync template issues

### ISSUE THREAD

**Issue by RandomDept** [microsoft/coe-starter-kit#8012]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hello all, hoping you can assist.

It looks like the current flows (Admin | Sync Template XYZ) aren't working in my environment at the moment. It all started when I was looking for an app and creator in the Admin - App permission centre.

I looked at the flows and had Admin - SyncTemplate V4 (Apps) as turned off.  After enabling I had an error in the Flow:

body.boolean' is no longer present in the operation schema. It should be removed before the workflow is re-saved.

I removed the 'false' from the run child flow section and the error went away. 

The v4 Apps flow now runs and shows as success, but nothing is being updated.  (even when trying manually)

The troubleshoot further, I wanted to see what child flow was being run in the V4 Apps Flow.
I copied the GUID and could not find any other Flows with this GUID. Odd, Does anyone know what Flow the v4 Apps calls?

Thank you.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.23

### What app or flow are you having the issue with?

Admin | SyncTemplate v4 Apps

### What method are you using to get inventory and telemetry?

Cloud flows

AB#2833

**Comment by Jenefer-Monroe** [comment]
It sounds like you might be in a broken state right now. 
Can you please share a screenshot like this so we can see what solutions you have installed in this environment? Be sure to show all records (all rows) and expand the indicated columns so that we can see full names
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/8dd512cc-21fc-4c64-adab-0f64c29b9bf2)

**Comment by geoff-lennon** [comment]
Hi Jenefer,

I'm encountering a similar issue (not seeing newly created apps / flows, etc. after a certain date in March) - should I enter a new question or participate in this thread?

Please see below for the requested screenshot from my environment.  Happy to provide any other details you might need to assist with troubleshooting.

Very much appreciate your help.

Thanks!

Geoff

***

![COE Screenshot](https://github.com/microsoft/coe-starter-kit/assets/80756413/c569860f-0229-40f9-a915-eb6f136a8922)

**Comment by Jenefer-Monroe** [comment]
Please post another issue, this person is experiencing issues where flows wouldnt turn on. So unless you are experiencing that too, please pose separately.

**Comment by RandomDept** [comment]
Thank you Jennifer,

Screenshot attached as requested.

![CurrentState](https://github.com/microsoft/coe-starter-kit/assets/151495228/3494ca82-f802-414d-b2c0-9b0336720283)

**Comment by Jenefer-Monroe** [comment]
Please go to the Admin Command Center > CoE Flows > Inventory. Validate that all flows are turned on and that there are no unmanaged layers as shown here. 
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/598b5024-2e1a-4390-a6b2-49644368c246)

**Comment by RandomDept** [comment]
Thanks again Jennifer,

Most are off, I'll go in one by one and activate. It also shows Apps v4 with an unmanaged layer. Is it safe to remove this / and or how :)?

![flowsoff](https://github.com/microsoft/coe-starter-kit/assets/151495228/bf39da7d-1de4-4fba-9efc-c4955af67ba9)

**Comment by Jenefer-Monroe** [comment]
Yes it looks like your inventory has not been running for all those offs. Please do remove the unmanaged layer.
There is a button at the top of that screen which will take you where you do this for the flows that have them. 
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/3dec841e-8cab-49f9-be76-cf3d62c4b1f2)

For more information on these please see [Remove unmanaged layers from inventory flows](https://learn.microsoft.com/power-platform/guidance/coe/after-setup#remove-unmanaged-layers-from-inventory-flows)

**Comment by RandomDept** [comment]
Thanks again Jenefer,

When I click the Remove solution layers button with the v4 Apps selected I get this error:

![image](https://github.com/microsoft/coe-starter-kit/assets/151495228/d8624a5b-1ee9-4951-bceb-8d443dfcf18a)

**Comment by Jenefer-Monroe** [comment]
I don't understand what state you are in. You can try manually deleting the solution layer instead. Please follow the steps outlined here [Removing other customizations](https://learn.microsoft.com/power-platform/guidance/coe/after-setup#removing-other-customizations)

**Comment by RandomDept** [comment]
Thank you Jenefer,

If I try to manually remove, It doesn't come up with the remove button. I am logged in as system admin with full admin rights.

![image](https://github.com/microsoft/coe-starter-kit/assets/151495228/c885bfff-2149-4512-b057-9700fd2dc7ae)

**Comment by RandomDept** [comment]
Also getting CommandCenterApp>SetCoEFlowsState.Run failed: { "error": { "code": "WorkflowTriggerIsNotEnabled", "message": "Could not execute workflow '51e2969a-6cf5-3077-c7aa-a8efb0621b6f' trigger 'manual' with state 'Disabled': trigger is not enabled." } } when trying to enable some of the Inventory items for the COE flows eg Admin | Sync Template v3 (Solutions)

**Comment by Jenefer-Monroe** [comment]
What do you see up herE? 
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/51ec9a1a-7e56-4696-ab87-d263979cc92c)

**Comment by Jenefer-Monroe** [comment]
Wait what type of envt is your CoE installed in? This can be found in the product admin pages: https://admin.powerplatform.microsoft.com/
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/4c23e8c8-2f8b-4d8b-af43-79a9f289be69)

**Comment by RandomDept** [comment]
> What do you see up herE? ![image](https://private-user-images.githubusercontent.com/89584709/320664441-51ec9a1a-7e56-4696-ab87-d263979cc92c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI2MTU2MzYsIm5iZiI6MTcxMjYxNTMzNiwicGF0aCI6Ii84OTU4NDcwOS8zMjA2NjQ0NDEtNTFlYzlhMWEtN2U1Ni00Njk2LWFiODctZDI2Mzk3OWNjOTJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA4VDIyMjg1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY0Yzc1NTNmN2UwZTgwYjUwYTMzNTU3OTQ5YjM1OWE1YjExNjc0OTVlMTljYTBiZTZhOWZiYWY1ZTAxMTU1MDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MMvMM6cjXwkYD5fZSgNq7kvSKGrRFRjwNac2TqIjF6Q)

Would help if I was in PowerAutomate instead of Apps :) my bad. I was able to remove the unmanaged layer.

**Comment by RandomDept** [comment]
> Wait what type of envt is your CoE installed in? This can be found in the product admin pages: https://admin.powerplatform.microsoft.com/ ![image](https://private-user-images.githubusercontent.com/89584709/237367659-4c23e8c8-2f8b-4d8b-af43-79a9f289be69.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI2MTU4MjIsIm5iZiI6MTcxMjYxNTUyMiwicGF0aCI6Ii84OTU4NDcwOS8yMzczNjc2NTktNGMyM2U4YzgtMmY4Yi00ZDhiLWFmNDMtNzlhOWYyODliZTY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDA4VDIyMzIwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1ZWIyOTA2YjFhMThlZDc4ZGVmNTEyODQ3YmJmZjI4NzI4OGU5MGJmOWUyY2M0MDE1MTM5NmNkYTdlZGQ5ODkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.O-GuTcCf4BIXU6MIQCEjRkk2SKMGuqKV29J2Iqfe7Uk)
![image](https://github.com/microsoft/coe-starter-kit/assets/151495228/b60a389c-1273-4857-a2b3-30c37005728b)

**Comment by RandomDept** [comment]
Thank you for your continued patience Jenefer,

I am now seeing the Admin | Sync Template V4 (Apps) as failed in the CoE Flows Inventory screen

![image](https://github.com/microsoft/coe-starter-kit/assets/151495228/0696409a-bc94-47fc-9b2f-88bc2a7a9fb6)

When I open the flow, and click Flow checker there are no errors and I can see this:

<img width="535" alt="image" src="https://github.com/microsoft/coe-starter-kit/assets/151495228/bdbfa2dd-08f1-456e-a4f0-2f766466136c">

**Comment by Jenefer-Monroe** [comment]
Yes I dont think we're out of the woods here yet. Let's ensure you are good to use the setup wizard, and then have you launch that to ensure all the correct flows are turned on. 

Please go make sure these four flows are turned on. And if not turn them on. 

- SetupWizard>CreateGroup
- SetupWizard>GetCurrentEnvironment
- SetupWizard>GetTenantID
- SetupWizard>GetUserDetails
 
Sometimes this one will fail to turn on due to a bad connection: **_SetupWizard>GetUserDetails_**

This usually means when the **_HTTP with Microsoft Entra ID (preauthorized)_** connection is broken or misconfigured.
Please go to connections and make sure the connection for these is as follows (or your tenant equivalent if you are not a commercial tenant: https://graph.microsoft.com/
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/f49797b8-0f48-4884-a026-2580bbc8b96f)
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/a31b22b8-bc81-4d81-a5b7-80dc740f2b82)

Sometimes the product creates one of these connected to the envt and this will not work.

**Comment by RandomDept** [comment]
Thank you,

All enabled without issue. Do you want me to re-run the CoE Setup and Upgrade Wizard?

**Comment by Jenefer-Monroe** [comment]
Yes now that we've hopefully gotten you unblocked please do run the setup wizard. Note that this is a tool for you to use both on clean installs as well as on upgrades as it will ensure all new flows added that release are turned on, and all new env vars are filled with good default values for your tenant.

**Comment by RandomDept** [comment]
Thank you so much, This is all now working.
